src/main/java/org/molgenis/calibratecadd/support/GavinUtils.java

```diff
@@ -54,6 +64,12 @@ private GavinUtils()
 		return geneToEntry;
 	}
 
+	private static Double getValueFromLine(String line, int index)
+	{
+		String[] split = line.split("\t");
+		return split[index].isEmpty() ? null : Double.parseDouble(split[index]);
```

Comment:
Throw an exception that is understandable to the user in case the String can't be parsed into a double (for example when you've added a trailing space).